inspirehep/modules/records/receivers.py

```diff
@@ -388,3 +388,14 @@ def generate_name_variations(sender, json, *args, **kwargs):
                 'output': full_name,
                 'payload': {'bai': bais[0] if bais else None}
             }})
+
+
+def populate_author_count(sender, json, *args, **kwargs):
```

Comment:
This needs to be added above to `enhance_after_index`: there's no magic here that collects the functions in this module and runs them, it's all very explicit...

Reply:
Oops, forgot about that. Will amend.